src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static Ball getRandomBall() {
+        int randomNumber = new Random().nextInt(100);
+        Ball ball = new Ball(ColorSupplier.getRandomColor(), randomNumber);
+        return ball;
```

Comment:
You can directly return the new `Ball` object without assigning it to a temporary variable `ball`.